modules/xmpp/SDPUtil.js

```diff
@@ -588,7 +588,10 @@ const SDPUtil = {
         }
         if (payloadType) {
             const payloadTypes
-                = videoMLine.payloads.split(' ').map(p => parseInt(p, 10));
+                = videoMLine.payloads
+                    .toString()
```

Comment:
i'd put a comment about the issue you found here that requires the `toString` call